src/main.ts

```diff
@@ -0,0 +1,89 @@
+import { Command, Helper } from 'dojo-cli/interfaces';
+import { Argv } from 'yargs';
+import * as fs from 'fs';
+import * as path from 'path';
+const DtsCreator: any = require('typed-css-modules');
+const read: any = require('read-file-stdin');
+const globby: any = require('globby');
+const async: any = require('async');
+const mkdirp: any = require('mkdirp');
+const chalk: any = require('chalk');
+
+interface TypingsArgs extends Argv {
+	out: string;
+}
+
+function processFile(args: TypingsArgs, creator: any, file: string, done: Function): void {
+	const outputPath: string = path.join(args.out, path.basename(file) + '.d.ts');
+
+	function execute(declarations: string, done: Function): void {
+		creator.create('', declarations).then(function (result: any) {
+			done(null, result);
+		}).catch(done);
+	};
+
+	function write(name: string, content: any, done: Function): void {
+		async.parallel([
+			async.apply(function(name: string, content: string, done: Function) {
+				mkdirp(path.dirname(name), function(err: any) {
+					if (err) {
+						done(err);
+					}
+					else if (args.out) {
+						fs.writeFile(name, content, done);
+						console.log(chalk.green(name));
+					} else {
+						done();
+					}
+				});
+			}, name, content.formatted)
+		], done);
+	};
+
+	async.waterfall([
```

Comment:
we're using `async` `await` elsewhere in the `cli`. I believe it could be used effectively here as well.